osquery/remote/http_client.cpp

```diff
@@ -30,27 +29,6 @@ const std::string kProxyDefaultPort{"3128"};
 
 const long kSSLShortReadError{0x140000dbL};
 
-/** This class is used to convert boost::system_exception
- *  to std::system_exception, since on freebsd osquery is on boost-1.64.
```

Comment:
Is this no longer relevant?

Reply:
Yes, this is not relevant since osquery code is based upon boost >= 1.66